clients/drcachesim/simulator/cache_simulator.h

```diff
@@ -65,6 +67,10 @@ class cache_simulator_t : public simulator_t {
     uint64_t
     remaining_sim_refs() const;
 
+    // Expose to allow clients to use cache configuration.
```

Comment:
Maybe worth mentioning that the returned pointer is only valid for the lifetime of this cache_simulator_t object

Reply:
Done.